build/android/gyp/jar.py

```diff
@@ -62,6 +62,11 @@ def main():
       help='List of .class file patterns to exclude from the jar.')
   parser.add_option('--stamp', help='Path to touch on success.')
 
+  parser.add_option(
+      '--jar-bin',
+      default='jar',
+      help='The jar binary. It empty, the jar binary is resolved from PATH.')
```

Comment:
typo: 'It' -> 'If'

(this typo is also repeated in the flags added to javac.py)

Reply:
done